2.1_first_module.ipynb

```diff
@@ -199,26 +286,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[\u001b[35minfo\u001b[0m] [0.000] Elaborating design...\n",
+      "[\u001b[35minfo\u001b[0m] [0.032] Done elaborating.\n",
+      "file loaded in 7.75056E-4 seconds, 5 symbols, 2 statements\n",
+      "test Passthrough Success: 0 tests passed in 2 cycles in 0.003979 seconds 502.64 Hz\n",
+      "x = ()\n",
+      "SUCCESS!!\n"
+     ]
+    }
+   ],
    "source": [
     "// Scala Code: Calling Driver to instantiate Passthrough + PeekPokeTester and execute the test.\n",
     "// Don't worry about understanding this code; it is very complicated Scala.\n",
     "// Think of it more as boilerplate to run a Chisel test.\n",
-    "val testResult = Driver(() => new Passthrough()) {\n",
-    "  c => new PeekPokeTester(c) {\n",
-    "    poke(c.io.in, 0)     // Set our input to value 0\n",
-    "    expect(c.io.out, 0)  // Assert that the output correctly has 0\n",
-    "    poke(c.io.in, 1)     // Set our input to value 1\n",
-    "    expect(c.io.out, 1)  // Assert that the output correctly has 1\n",
-    "    poke(c.io.in, 2)     // Set our input to value 2\n",
-    "    expect(c.io.out, 2)  // Assert that the output correctly has 2\n",
-    "  }\n",
+    "test(new Passthrough()) { c =>\n",
+    "    c.io.in.poke(0.U)     // Set our input to value 0\n",
+    "    c.io.out.expect(0.U)  // Assert that the output correctly has 0\n",
+    "    c.io.in.poke(1.U)     // Set our input to value 1\n",
+    "    c.io.out.expect(1.U)  // Assert that the output correctly has 1\n",
+    "    c.io.in.poke(2.U)     // Set our input to value 2\n",
+    "    c.io.out.expect(2.U)  // Assert that the output correctly has 2\n",
```

Comment:
Do we have an example anywhere of an `expect` that fails? Because the current system doesn't return a test success / failure value, it might be useful to demonstrate what happens (exception), and maybe also have a side note on integration w/ ScalaTest